maker-core/list-of-scopes.md

```diff
@@ -1,32 +1,14 @@
 # List of Scopes
 
-The Endgame Scopes are divided into three categories, Primary, Supporting, and Governance.
+{% hint style="warning" %} This documentation describes ongoing changes in functionality and processes. Be aware that parts may be inaccurate or out of date. {% endhint %}
 
-{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+Here is a list of the five Scopes:
```

Comment:
```suggestion
The Maker Protocol has five Scopes. They are:
```

Reply:
We're being inconsistent with the capitalization of "scopes".

"Scopes" seems better